src/main/java/com/training/tinyurl/security/service/UserDetailsServiceImpl.java

```diff
@@ -18,10 +18,9 @@ public class UserDetailsServiceImpl implements UserDetailsService {
 
     @Override
     public UserDetails loadUserByUsername(String email) throws UsernameNotFoundException {
-        Optional<UserInfoEntity> user = userInfoDb.findById(email);
-        if(user.isPresent()){
-            return new AppUserDetails(user.get());
-        }
-        throw new UsernameNotFoundException("No record found for "+email);
+        String finalEmail = email.toUpperCase();
```

Comment:
What is final email ?
If final, it should be
`final String emailInCaps = email.toUpperCase();`